src/dev-gui.ts

```diff
@@ -48,11 +48,6 @@ class DevGUI {
   }
 
   reset(): void {
-    // The lil-gui bundled with the current three.js version does not have the `onOpen` method to save the open folders
-    // as they are individually opened. This is good enough in between different changes to the watched objects. The
-    // last change will not persist between page reloads unless some event triggered this reset method.
-    this.saveOpenFolders();
-
```

Comment:
Not relying on three.js bundled version allows us to keep ownership of what version we want to use. I could have split this change in a separate PR, but it was easier to bundle that in.